en/docs/develop/product-apis/devportal-apis/devportal-v1/devportal-v1.yaml

```diff
@@ -364,7 +364,7 @@ paths:
       summary: |
         Generate a SDK for an API
       description: |
-        This operation can be used to generate SDKs (System Development Kits), for the APIs available in the API Store, for a requested development language.
+        This operation can be used to generate SDKs (System Development Kits), for the APIs available in the API Devportal, for a requested development language.
```

Comment:
Let's use "Developer Portal"

Reply:
fixed this in 3.0.0, 3.1.0 and 3.2.0 PRs with https://github.com/wso2/docs-apim/pull/2681/commits/2ae564882d179dc629c6bec5a5df6b145732fed8, https://github.com/wso2/docs-apim/pull/2683/commits/02c76cf539184a55c03a5e4e4a195461f35e3f7b, https://github.com/wso2/docs-apim/pull/2684/commits/2f79661f87504c780784a81be68fc2cd36a1a3e8